wgengine/userspace.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/tailscale/wireguard-go/tun"
 	"github.com/tailscale/wireguard-go/wgcfg"
 	"go4.org/mem"
+	"inet.af/netaddr"
```

Comment:
btw, I just added https://godoc.org/inet.af/netaddr#IP.As4 for you if that'll help.

Reply:
I was just about to comment about that being useful in a bunch of places. Thanks!